client/src/actions/__tests__/authActions.test.js

```diff
@@ -0,0 +1,145 @@
+import { makeMockStore } from "../../test/testUtils";
+import authActions from "../authActions";
+import jwt_decode from "jwt-decode";
+import API from "../../lib/api";
+import setAuthToken from "../../utils/setAuthToken";
+
+const store = makeMockStore({
+  auth: {
+    isAuthenticated: false,
+    user: {},
+    loading: false
+  }
+});
+
+const mockLoginData = {
+  email: "jenniferlynparsons@gmail.com",
+  password: "boogadoo1"
+};
+
+const mockRegisterData = {
+  name: "Jennifer",
+  email: "littleoracle@gmail.com",
+  password: "boogadoo1",
+  password2: "boogadoo1"
+};
+
+const mockHistory = {
+  length: 10,
+  action: "PUSH",
+  location: { pathname: "/dashboard", search: "", hash: "", key: "zh4boo" },
+  push: jest.fn()
+};
+
+const mockResponse = {
+  success: true,
+  token:
+    "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjVjNjMxM2E0YzMxOGJiNjIyOThiMjNkNCIsIm5hbWUiOiJKZW5uaWZlciIsImlhdCI6MTU1ODk3MTY4MSwiZXhwIjoxNTkwNTI4NjA3fQ.1Tn8FLJE2gUj0D0ka91e79cLEQz6ttSkQ0sZNd3GGE8"
```

Comment:
Just checking, is this a real auth token? Hopefully you're not checking in possibly sensitive information.

Reply:
i was real, but it's for a garbage account that's been deleted so no longer exists.